sway-core/src/ir_generation/function.rs

```diff
@@ -5520,27 +5520,37 @@ pub fn get_memory_representation(ctx: &Context, t: Type) -> MemoryRepresentation
                 MemoryRepresentation::Array(Box::new(item), *len)
             }
         }
-        TypeContent::Pointer => MemoryRepresentation::Blob { len_in_bytes: 8 },
+        TypeContent::Pointer | TypeContent::TypedPointer(_) => {
+            MemoryRepresentation::Blob { len_in_bytes: 8 }
+        }
```

Comment:
Nitpick. No need for curly brackets.
```suggestion
        TypeContent::Pointer
        | TypeContent::TypedPointer(_) => MemoryRepresentation::Blob { len_in_bytes: 8 },
```

Reply:
Done.